python-service/src/app.py

```
@@ -71,4 +71,4 @@ def review_function():
 
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=8081, debug=False)
+    app.run(host="0.0.0.0", port=8081, debug=True)
```

Ruff (0.15.20)

[error] Possible binding to all interfaces

(S104)